### DISCUSSION TITLE
【委托】报酬60菲林！

### DISCUSSION THREAD

**Discussion by WLMRMZ** [share121/inter-knot#820] [其他]
明天就要军训了，求下雨教程或者有无绳匠愿意接的，报酬60菲林，
![Uploading 微信图片编辑_20240810171609.jpg…]()
本人在光映广场【HIA】俱乐部在线等，急！！！！！！！！！

**Comment by EF-LY** [comment]
-.--..-...--.-./-..------.---.-/-..-...---..--.-！

**Comment by moyudashi66** [comment]
这个委托我接，什么，军训？😅算了我接不了，，你先军训去吧，我先当将军去了

**Reply by WLMRMZ** [reply]
嗯呢，嗯呢呢（生气气）

**Comment by Lanxey** [comment]
能接受其他方式躲开军训吗？比如不小心往你饭菜里倒半斤泻药

**Reply by WLMRMZ** [reply]
哦！我的上帝，这个想法真是糟糕透了！

**Comment by SDCOM-0415** [comment]
这个委托我接了，赛博帮你军训！

**Reply by WLMRMZ** [reply]
嗯呢（你的好意我领了）

**Comment by crystal-blood** [comment]
建议去找隔壁纳维莱特看琪亚娜的故事让他在光映广场哭就可以了，可操作性极强@纳维莱特

**Reply by WLMRMZ** [reply]
成分复杂.jpg

**Comment by Magic-gif** [comment]
难度很高，实在不行的话人工降雨吧！

**Reply by WLMRMZ** [reply]
我会想办法向天上发射干冰的！

**Comment by sky27950** [comment]
太元浩师雷火精，结阴聚阳守雷城。
关伯风火登渊庭，作风兴电起幽灵。
飘诸太华命公宾，上帝有敕急速行。
收阳降雨顷刻生，驱龙掣电出玄泓。
我今奉咒急急行，此乃玉帝命君名，敢有拒者罪不轻。
急急如律令。

**Reply by WLMRMZ** [reply]
谢谢大师，我悟了！

**Comment by WLMRMZ** [comment]
各位，各方法我会在军训时一一尝试，成功后会将60菲林打款到各位账户上，谢谢了！（对身体有害的暂不尝试哦~）( •̀ ω •́ )✧

**Comment by keerdun** [comment]
我现在就去天恒山帮你崩龙王庙！

**Comment by kaijinti** [comment]
我去，楼上道爷来真的？

**Comment by shuyishaoxiancao** [comment]
这个委托我接了

**Comment by yemoyuxiao** [comment]
很简单，让那维莱特玩崩三就行

[the remark this answers is not in the document]
**Reply by SeeleVollereid** [reply]
因为大号没了@Seele1st
